x-pack/plugins/data_usage/server/types/rest_types/usage_metrics.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { schema, type TypeOf } from '@kbn/config-schema';
+
+const METRIC_TYPE_VALUES = [
```

Comment:
@neptunian I chose these values from the tech. design document. Feel free to suggest changes.